bancor_research/bancor_simulator/v3/spec/state.py

```diff
@@ -1103,41 +1095,43 @@ def get_bnt_trading_liquidity(state: State, tkn_name: str) -> Decimal:
     return state.tokens[tkn_name].bnt_trading_liquidity.balance
 
 
-def get_standard_reward_providers(state: State, id: int) -> list:
+def get_standard_reward_pool_token_name(state: State, id: int) -> str:
     """
-    Get the standard reward providers for a given program id.
+    Get the standard reward pool token name for a given id.
     """
-    return state.standard_reward_programs[id].providers
-
-
-def get_standard_reward_tkn_name(state: State, id: int) -> Tuple[str, str]:
-    """
-    Get the standard reward tkn_name for a given id.
-    """
-    return (
-        state.standard_reward_programs[id].tkn_name,
-        state.standard_reward_programs[id].rewards_token,
-    )
+    return get_pooltoken_name(state.standard_reward_programs[id].tkn_name)
 
```

Comment:
@mikewcasale : above is just poor comparison:
- Function `get_standard_reward_providers` was removed
- Function `get_standard_reward_tkn_name` was slightly modified, and renamed to `get_pooltoken_name`